workflows/theiacov/wf_theiacov_clearlabs.wdl

```diff
@@ -12,14 +12,15 @@ import "../../tasks/species_typing/task_pangolin.wdl" as pangolin
 import "../../tasks/gene_typing/task_sc2_gene_coverage.wdl" as sc2_calculation
 import "../../tasks/quality_control/task_qc_check_phb.wdl" as qc_check
 import "../../tasks/task_versioning.wdl" as versioning
+import "../../workflows/utilities/wf_organism_parameters.wdl" as set_organism_defaults
 
 workflow theiacov_clearlabs {
   meta {
     description: "Reference-based consensus calling for viral amplicon ont sequencing data generated on the Clear Labs platform."
   }
   input {
     String samplename
-    File clear_lab_fastq
+    File read1
     String organism = "sars-cov-2"
```

Comment:
I'm glad that this is kept as default, but most all other default input variables (specific to sars-cov-2) are now in the organism parameters subwf. Soooo nice